extension-testcontainer/src/main/java/io.camunda.zeebe.process.test.extension.testcontainer/ZeebeProcessTestExtension.java

```diff
@@ -78,10 +81,14 @@ public void beforeAll(final ExtensionContext extensionContext) {
   @Override
   public void beforeEach(final ExtensionContext extensionContext) {
     final Object engineContent = getStore(extensionContext.getParent().get()).get(KEY_ZEEBE_ENGINE);
+    final ObjectMapper customObjectMapper = getCustomMapper(extensionContext);
```

Comment:
🔧 Let's work with `Optionals` instead of returning a `null` if the custom object mapper is not available.

Same goes for the other extension of course.

Reply:
Nice suggestion ❤️